telemetry/specs/INT.mdk

```diff
@@ -992,11 +1010,25 @@ each bit corresponds to a specific standard metadata as specified in Section 3.
   Details of the metadata semantics YANG model can be accessed at the link below:
   https://github.com/p4lang/p4-applications/blob/master/telemetry/code/models/p4-dtel-metadata-semantics.yang
   
-  Each instruction requests 4 bytes of metadata to be inserted at each hop,
-  except if bit 6 is set, which requires 8 bytes of metadata. Per-hop
-  metadata length is set accordingly at the INT source.
-  
+  Bits 0 - 13 are Baseline INT Instructions and Bit 14, Domain Specific Instruction.
```

Comment:
"Bit 14 is Domain ..."

Reply:
minor nit, let's correct.

Reply:
Removed Bit 14 and added text to address Domain Specific processing